apps/calcom/docker-compose.yml

```diff
@@ -0,0 +1,57 @@
+version: '3.8'
+
+services:
+
+  calcom:
+    container_name: calcom
+    image: calcom/cal.com:v3.7.9
+    restart: unless-stopped
+    ports:
+      - ${APP_PORT}:3000
+    networks:
+      - tipi_main_network
+    environment:
+      - DATABASE_HOST=${POSTGRES_HOST}
```

Comment:
We should add a new Postgres container in this file instead of re-using tipi's Postgres db

Reply:
Curious as why you chose this approach? Is there any issue in having multiple databases with a single postgres instance? I would have thought it would be preferable to share the postgres instance since it will be used by many apps and otherwise users could easily end up running 10 instances of postgres which might degrade performance?

Reply:
For a few reasons, the most important one would be that we'd have to lock ourselves in a particular version of Postgres that is compatible with the app. If some apps don't update the PG version we would be stuck with this old version with Tipi as well. The second reason is for backups, it would be impossible for a user to extract an app and its data without taking everything or writing a script to extract the needed data and re-import it in another DB.